test/test_cost.py

```diff
@@ -920,7 +937,11 @@ def test_ddpg_tensordict_keys(self, td_est):
             value,
             loss_function="l2",
         )
-        key_mapping = {"state_action_value": ("value", "state_action_value_test")}
+        key_mapping = {
+            "state_action_value": ("value", "state_action_value_test"),
+            "reward": ("reward", "reward_test"),
+            "done": ("done", "done_test"),
```

Comment:
Why not having reward_test and done_test nested? Like this we make sure that we cover the nested keys too.
eg:
```python
("test", "reward_test")
```